pulp/hero-target.c

```diff
@@ -100,15 +101,17 @@ hero_dma_memcpy_async(void *dst, void *src, int size)
       pulp_tryread((unsigned *)ext_addr_tmp);
     pulp_tryread((unsigned *)((ext_addr + size_tmp - 1) & 0xFFFFFFFC));
 
-    // just wait for the last one...
-    dma = (hero_dma_job_t)plp_dma_memcpy_priv(ext_addr,loc_addr,size_tmp,ext2loc);
+    //dma_job = (hero_dma_job_t)plp_dma_memcpy_priv(ext_addr,loc_addr,size_tmp,ext2loc);
+    dma_cmd = plp_dma_getCmd(ext2loc, size, PLP_DMA_1D, PLP_DMA_TRIG_EVT, PLP_DMA_NO_TRIG_IRQ, PLP_DMA_PRIV);
```

Comment:
I am not sure if this change is sufficient to fix the issue described in #8 . This should definitely be tested. @ctbur could you have a look please?

Reply:
Yes, I will test it as soon as my other DMA and compiler issues are resolved. This way I won't have to create a custom test case.